src/components/User/Assets/SystemAssets/components/AssetManager.vue

```diff
@@ -353,7 +375,7 @@
               v-model="asset.creatorDetails.stellar.assetCode"
```

Comment:
This coment not for this place(364)
{{ assetType.label }} - > {{ assetType.translationId | globalize }}

Reply:
done